Makefile

```diff
@@ -0,0 +1,7 @@
+# Main makefile for the testing framework
```

Comment:
Maybe this comment should be expanded to say this is intended to be executed by the actual project containing the tests.